src/main/kotlin/com/charlatano/game/hooks/EntityIteration.kt

```diff
@@ -67,4 +91,5 @@ fun constructEntities() = every(512) {
 			if (!contains(context)) add(context)
 		}
 	}
+    state = SignOnState[CSGO.csgoEXE.int(clientState + EngineOffsets.dwSignOnState)]
```

Comment:
This was suppose to be at the start of the function but end up here after diff i'm not sure why. Doesn't change the outcome tho.